spring-components/spring-boot-sparql-web/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.rdf4j</groupId>
+		<artifactId>rdf4j-spring-boot-components</artifactId>
+		<version>3.7.0</version>
+	</parent>
+	<properties>
+		<spring.boot.version>2.4.3</spring.boot.version>
```

Comment:
Given that we have this property in the parent pom, I think it should not be needed here